packages/cli/src/utils/prompt/utils/deepExtend.ts

```diff
@@ -1,5 +1,5 @@
 // Taken from https://github.com/unclechu/node-deep-extend/blob/master/lib/deep-extend.js
-// es2017-ified by Tim Suchanek, now it's about 2.5 times faster
+// es2017-Simplified by Tim Suchanek, now it's about 2.5 times faster
```

Comment:
I think this was intentional - it should mean "made es2017"

Reply:
Oh, my bad 😅  ~!  Reverted it!